model-optimizer/extensions/front/tf/non_max_suppression_normalize.py

```diff
@@ -64,18 +64,34 @@ def replace_sub_graph(graph: Graph, match: dict, **kwargs):
                                                                 {'name': nms.soft_get('name') + '/Unsqueeze_1'})
         nms.in_port(1).get_connection().insert_node(unsqueeze_box_scores)
 
-        # prepare output
-        crop_box_indices = Crop(graph, {'name': nms.soft_get('name') + '/Crop', 'axis': int64_array([1]),
+        nms_name = nms.soft_get('name', nms.id)
```

Comment:
Rewrite this transformation to not use the `pattern` function.

Reply:
Done.